WebRoot/js/zimbraMail/prefs/view/ZmFilterRuleDialog.js

```diff
@@ -1389,7 +1389,11 @@ function(ev) {
 		if (msg = this._checkCondition(c)) {
 			break;
 		} else {
-			rule.addCondition(c.testType, c.comparator, c.value, c.subjectMod, c.caseSensitive);
+			if (c.testType === ZmFilterRule.C_TEST_MAP[ZmFilterRule.C_DATE]) {
+				rule.addCondition(c.testType, c.comparator, Number(c.value), c.subjectMod, c.caseSensitive);
+			} else {
+				rule.addCondition(c.testType, c.comparator, c.value, c.subjectMod, c.caseSensitive);
+			}
```

Comment:
Does a client need to send the date in long format as Number explicitly? 
Before the change, it is sent as String. Doesn't the server automatically convert it from a string it to a number?